scorecard.html

```diff
@@ -0,0 +1,135 @@
+      <div class="bottom" id="4"></div>
+      <div class="bonus"id="4"></div>
+    </div>
+    <div class="frame">
+      <div class="frame-number">5</div>
+      <div class="top">
+        <div class="left" id="5"></div>
+        <div class="right" id="5"></div>
+      </div>
+      <div class="bottom" id="5"></div>
+      <div class="bonus"id="5"></div>
+    </div>
+    <div class="frame">
+      <div class="frame-number">6</div>
+      <div class="top">
+        <div class="left" id="6"></div>
+        <div class="right" id="6"></div>
+      </div>
+      <div class="bottom" id="6"></div>
+      <div class="bonus"id="6"></div>
+    </div>
+    <div class="frame">
+      <div class="frame-number">7</div>
+      <div class="top">
+        <div class="left" id="7"></div>
+        <div class="right" id="7"></div>
+      </div>
+      <div class="bottom" id="7"></div>
+      <div class="bonus"id="7"></div>
+    </div>
+    <div class="frame">
+      <div class="frame-number">8</div>
+      <div class="top">
+        <div class="left" id="8"></div>
+        <div class="right" id="8"></div>
+      </div>
+      <div class="bottom" id="8"></div>
+      <div class="bonus"id="8"></div>
+    </div>
+    <div class="frame">
+      <div class="frame-number">9</div>
+      <div class="top">
+        <div class="left" id="9"></div>
+        <div class="right" id="9"></div>
+      </div>
+      <div class="bottom" id="9"></div>
+      <div class="bonus"id="9"></div>
+    </div>
+    <div class="frame">
+      <div class="frame-number">10</div>
+      <div class="top">
+        <div class="left" id="10"></div>
+        <div class="right" id="10"></div>
+      </div>
+      <div class="bottom" id="10"></div>
+      <div class="bonus"id="10"></div>
+    </div>
+    <div class="frame" >
+      <div class="frame-number">Bonus</div>
+      <div class="score" id="bonus"></div>
+    </div>
+    <div class="frame" >
+      <div class="frame-number">Score</div>
+      <div class="score" id="main"></div>
+    </div>
+    <div class="frame">
+      <div class="frame-number">Total</div>
+      <div class="score" id="total"></div>
+    </div>
+    <div class="pins">
+      <p>Select Pins</p>
+      <button class="pin" id="button-0" value=0 >0</button>
+      <button class="pin" id="button-1" value=1 >1</button>
+      <button class="pin" id="button-2" value=2 >2</button>
+      <button class="pin" id="button-3" value=3 >3</button>
+      <button class="pin" id="button-4" value=4 >4</button>
+      <button class="pin" id="button-5" value=5 >5</button>
+      <button class="pin" id="button-6" value=6 >6</button>
+      <button class="pin" id="button-7" value=7 >7</button>
+      <button class="pin" id="button-8" value=8 >8</button>
+      <button class="pin" id="button-9" value=9 >9</button>
+      <button class="pin" id="button-10" value=10 >10</button>
+    </div>
+    <h2>Game Over </h2>
+    <button id="restart">Start New Game</button>
+  </div>
+</body>
+</html>
```

Comment:
I like that you can click on the pins as input rather than entering the input - It is a good solution to limiting the user from entering too large numbers and also adds to the user experiences